lib/opensusebasetest.pm

```diff
@@ -610,9 +610,24 @@ sub handle_uefi_boot_disk_workaround {
     send_key_until_needlematch 'tianocore-select_opensuse_or_sles', 'up';
     save_screenshot;
     wait_screen_change { send_key 'ret' };
-    # efi file
-    send_key_until_needlematch 'tianocore-select_grubaa64_efi', 'up';
-    wait_screen_change { send_key 'ret' };
+    # efi file, first check shim.efi exist or not
+    my $counter        = 10;
+    my $shim_efi_found = 1;
+    while (!check_screen('tianocore-select_shim_efi', 2)) {
```

Comment:
Wei, 

I did check the recent builds. and found that the shim.efi was there whatever secureboot enabled or not.

IMO, send_key_until_needlematch 'tianocore-select_shim_efi', 'up'; should be enough now..

Reply:
oops, I forgot to consider the older OS version, omit my comments above